source/views/building-hours/report/editor.tsx

```diff
@@ -153,17 +178,17 @@ const ToggleButton = (props: ToggleButtonProps) => {
 	)
 }
 
-type DatePickerCellProps = {
+type DatePickerAccessoryProps = {
 	date: Moment
-	title: string
 	onChange: (date: Moment) => unknown
 }
 
-function DatePickerCell(props: DatePickerCellProps) {
+function DatePickerAccessory(props: DatePickerAccessoryProps) {
 	let format = 'h:mm A'
-
-	let accessory = (
+	return (
 		<DatePicker
+			displayAndroid="clock"
+			displayIos='inline'
```

Comment:
prettier?

Reply:
Yes thanks. I made a follow-up for more cleanup and this change is wrapped into:
- #6173